examples/ai-sdk/gpt-5.ts

```diff
@@ -0,0 +1,38 @@
+import { Agent, run, tool } from '@openai/agents';
+import { aisdk } from '@openai/agents-extensions';
+import { z } from 'zod';
+import { openai } from '@ai-sdk/openai';
+
+export async function main() {
+  const getWeatherTool = tool({
+    name: 'get_weather',
+    description: 'Get the weather for a given city',
+    parameters: z.object({ city: z.string() }),
+    execute: async ({ city }) => `The weather in ${city} is sunny`,
+  });
+  const agent = new Agent({
+    name: 'Helpful Assistant',
+    instructions:
+      'You are a helpful assistant. When you need to get the weather, you must use tools.',
+    tools: [getWeatherTool],
+    model: aisdk(openai('gpt-5-mini')),
+    modelSettings: {
+      providerData: {
+        providerOptions: {
```

Comment:
Just to clarify: I didn't make any changes to ai-sdk integration; Passing these options is the standard way for ai-sdk provider for OAI, so I added example code.